### ISSUE TITLE
Typo in Example in documentation for method channels.createChannel

### ISSUE THREAD

**Issue by yousafhanif6186** [gram-js/gramjs#80]
Hi, I was trying to use **channels.createChannel** method I found a typo in constructor name in Api class 
**Api.channels.createChannel**   the name in code is **CreateChannel** but in example, it's wrong.

https://painor.gitbook.io/gramjs/working-with-channels-and-supergroups-and-geogroups/channels.createchannel

**Comment by painor** [comment]
yes. all api endpoints are CamelCase. the docs were generated wrong. it will be fixed soon and switched to another site.
